saleor/graphql/product/resolvers.py

```diff
@@ -119,11 +124,11 @@ def resolve_products(
         qs = filter_products_by_attributes(qs, attributes)
 
     if categories:
-        categories = get_nodes(categories, Category)
+        categories = get_nodes(categories, "Category")
```

Comment:
`get_nodes` doesn't support lazy graphene type, maybe we could optionally take `info` to resolve it? The changes were made because I need this resolver inside the types, otherwise I would have gotten a circular import.